src/ic-cdk-http-kit/src/storage.rs

```diff
@@ -0,0 +1,52 @@
+use crate::mock::{hash, Mock};
+use ic_cdk::api::management_canister::http_request::{
+    CanisterHttpRequestArgument, HttpResponse, TransformArgs,
+};
+use std::collections::HashMap;
+use std::sync::RwLock;
+
+thread_local! {
+    static MOCKS: RwLock<HashMap<String, Mock>> = RwLock::new(HashMap::new());
```

Comment:
thread_local! removes the need for a thread-safe cell type like RwLock, and a thread-local cell type like RefCell should be used instead.

Reply:
Done.